src/Appwrite/Auth/OAuth2/Disqus.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Appwrite\Auth\OAuth2;
+
+use Appwrite\Auth\OAuth2;
+
+// Reference Material
+// https://disqus.com/api/docs/auth/
+
+class Disqus extends OAuth2
+{
+    private string $endpoint = 'https://disqus.com/api/';
+    protected array $user = [];
+    protected array $tokens = [];
+    protected array $scopes = [
```

Comment:
it contains 4 scopes , `read`, `write`, `email`, `admin`. 

On further evaluation, I found only `read` and `email` neccessary

Reply:
Thanks for checking ✅